queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon';
```

Comment:
* Kindly fix this query `select * from animals where name like '%mon';` as the result i get is not right 
![image](https://user-images.githubusercontent.com/52497246/177217795-5e8dd50f-57d3-4035-bfeb-a6169cbf13ad.png)